automation-for-stable-branches/README.md

```diff
@@ -0,0 +1,66 @@
+# Automation for moving the stable branches
+
+Every Friday we move the stable branches of our repositories so that new `prod` images are rebuilt
+for Packit Service. There are still more and more repositories which
+we need to move the stable branches for and this work is repetitive and can be done automatically.
+
+Repositories which we currently need for Packit Service:
+
+- packit
+- packit-service
+- packit-service-fedmsg
+- packit-service-centosmsg
+- sandcastle
+- dashboard
+- tokman
+
+- ogr (at the moment we have only master branch for it,
+  but we planned to create a stable branch for it as well)
+
+Dependencies (order of moving):
+
+- [packit -> packit-service](https://github.com/packit/packit-service/blob/a1ba5988ce2a2409b4f52f05324587303b52e676/files/install-deps-worker.yaml#L50)
+- [ogr -> packit-service](https://github.com/packit/packit-service/blob/a1ba5988ce2a2409b4f52f05324587303b52e676/files/install-deps-worker.yaml#L52)
+
+AC:
+
+- have a mechanism to determine whether stage works fine
+  and move the stable branches to appropriate commit hashes
```

Comment:
this is a really good point!

since this task is about automation, I'd make this step in way that "a human approves" all the latest changes to be pushed to stable branches

not sure if there is an automated way to figure out that "stage is good" and we can update the stable branches